learning-guide/beginner.md

```diff
@@ -1,5 +1,35 @@
 # Beginner Guide
 
+## What You Need To Know
+
+In order to develop decentralized applications on Ethereum or other blockchains, you will need some fundamental skills. You may already have some of these skills. If so, that's great! If you are here though, it is likely at least some of this is new to you. Here are the essentials of what you need to know:
+
+1. How the Blockchain Works
+   
+   You will need to have a basic understanding of how the blockchain works. What is a private and public key? Wht is a block and how is it validated? What even is a blockchain. There are some great materials out there for you and we have gathered some of the best below. If books are your thing, you'll prababl want to start with "Mastering Bitcoin." This will give you some interesting history and teach you everything you need to know about Bitcoin.
+
+   You can then follow this up with "Mastering Ethereum". As a dapp developer specifically, you will need to know about smart contracts and the EVM. This will not only teach you about Ethereum, but also many of the concepts you will need for developing on other chains. 
```

Comment:
```suggestion
You can then follow this up with "Mastering Ethereum". As a Dapp developer specifically, you will need to know about smart contracts and the EVM. This will not only teach you about Ethereum but also many of the concepts you will need for developing on other chains. 
```